src/utils/exploreContent.js

```diff
@@ -44,6 +44,18 @@ export const EXPLORE_FILTER_OPTIONS = [
     type: 'network',
     default: true,
   },
+  {
+    name: 'Harmony',
+    value: 1666600000,
+    type: 'network',
+    default: true,
+  },
+  {
+    name: 'HarmonyTest',
```

Comment:
Does the test network need to be in here? I don't see it in the chains.js.

Reply:
Removed the testnet configuration from exploreContent